galley/pkg/server/server.go

```diff
@@ -219,6 +220,7 @@ func newServer(a *Args, p patchTable) (*Server, error) {
 	mcp.RegisterAggregatedMeshConfigServiceServer(s.grpcServer, s.mcp)
 	mcp.RegisterResourceSourceServer(s.grpcServer, s.mcpSource)
 
+	configz.Register(distributor)
```

Comment:
Can you explicitly create this as a custom topic and register through the ctrlz.Run command?

The reason I introduced the static register method was to deal with the fact that some of our existing code creates the MCP client way deep, away from where ctrlz.Run gets called from. It would be nice we can avoid adding more dependencies on static registration.